adminapp/src/pages/OrganizationDetailPage.jsx

```diff
@@ -29,6 +30,12 @@ export default function OrganizationDetailPage() {
           <RelatedList
             title={`Memberships (${model.memberships.length})`}
             rows={model.memberships}
+            addNewLabel="Create another membership"
+            addNewLink={createRelativeUrl(`/membership/new`, {
+              organizationId: model.id,
+              organizationLabel: `(${model.id}) ${model.name || "Name Unavailable"}`,
```

Comment:
We don't use this terminology anywhere else. And why would someone care that the name is unavailable? Use `-` or something more succint.

Reply:
Okay, can change this too, it looked weird empty, but didn't think about adding a symbol rather than wording.